index.html

```diff
@@ -718,24 +736,26 @@ <h3>Resource Timing Attributes</h3>
 <h2>Creating a resource timing entry</h2>
 <p>To <dfn export="">mark resource timing</dfn> given a
 [=/fetch timing info=] |timingInfo|, a DOMString
-|requestedURL|, a DOMString |initiatorType| and a <a>global object</a> |global|:
+|requestedURL|, a DOMString |initiatorType| a <a>global object</a> |global|, and a string |cacheMode|,
+perform the following steps:
 <ol>
 <li>Create a <a>PerformanceResourceTiming</a> object |entry| in |global|'s [=global object/realm=].
-<li><a>Setup the resource timing entry</a> for |entry|, given |initiatorType|, |requestedURL| and
-|timingInfo|.
+<li><a>Setup the resource timing entry</a> for |entry|, given |initiatorType|, |requestedURL|,
+|timingInfo|, and the empty string or "<code>local</code>" |cacheMode|.
```

Comment:
If we're just passing |cacheMode|, I think you can drop the "empty string or local" part